src/components/Design/design.module.scss

```diff
@@ -0,0 +1,20 @@
+.designContainer {
+  display: flex;
+}
+.fontBasestyle {
+  font-family: 'Dela Gothic One';
+  font-style: normal;
+  font-weight: 400;
+  letter-spacing: -0.04em;
+  line-height: 120%;
+  font-size: 32px;
+}
+
+.HeadingBasestyle {
+  font-family: Dela Gothic One;
+  font-style: normal;
+  font-weight: 400;
+  letter-spacing: -0.04em;
+  line-height: 120%;
+  color: '#000000';
+}
```

Comment:
1. should be `HeadingBaseStyle`
2. `font-family` should enclosed it in quotes
3. the default value of `font-style` is `normal` and `font-weight` is `400`, so don't need to be set
4. `color` don't need quotes

Reply:
Hi Double,

thanks for your detailed explanation. All issues have been fixed. Cheers.